### PR TITLE
cmd/bootcfg: Start HTTP and gRPC(optional) via bootcfg binary

### PR DESCRIPTION
**Pull request by dghubble**

### CODE REVIEW

**Comment by dghubble** [comment]
cc @sym3tri

File: cmd/bootcfg/main.go [hunk 1]
```diff
@@ -108,16 +113,35 @@ func main() {
...
+	})
+
+	// gRPC Server (feature hidden)
+	if flags.rpcAddress != "" {
+		grpcServer, err := rpc.NewServer(bootcfgServer)
+		if err != nil {
+			log.Fatal(err)
+		}
+		log.Infof("starting bootcfg gRPC server on %s", "127.0.0.1:8081")
+		lis, err := net.Listen("tcp", "127.0.0.1:8081")
```
Comment (sym3tri) [inline cmd/bootcfg/main.go]: Why not listen on `flags.rpcAddress`?

**Comment by sym3tri** [comment]
Otherwise changes LGTM.

I'd start looking into this... might save you some time.
https://coreos.com/blog/gRPC-protobufs-swagger.html

File: cmd/bootcfg/main.go (hunk 1 above)
Comment (dghubble) [inline cmd/bootcfg/main.go]: oh thanks, that was the entire point of adding rpcAddress :)

**Comment by dghubble** [comment]
With Go 1.6 we can multiplex HTTP and gRPC off the same port like in Brandon's example and I'll remove the optional `rpcAddress`. #103

**Pull request merged**